.github/workflows/release-binaries.yml

```diff
@@ -60,6 +60,8 @@ jobs:
       enable-pgo: ${{ steps.vars.outputs.enable-pgo }}
       release-binary-basename: ${{ steps.vars.outputs.release-binary-basename }}
       release-binary-filename: ${{ steps.vars.outputs.release-binary-filename }}
+      runs-on: ${{ steps.vars.outputs.runs-on }}
```

Comment:
It also looks like the `runs-on` options above haven't been updated for the depot runners?

Reply:
Selecting `ubuntu-22.04` will use the depot runners, see https://github.com/llvm/llvm-project/pull/117111/files#diff-07d8c704ff21e83c1d4e79f9f7bc1472dc35deac2cbadc8b53b59131e30879c1R149